prometheus_client/exposition.py

```diff
@@ -210,6 +217,7 @@ def start_wsgi_server(
         client_capath: Optional[str] = None,
         protocol: int = ssl.PROTOCOL_TLS_SERVER,
         client_auth_required: bool = False,
+        debug: bool = False
```

Comment:
Rather than just a debug flag, I wonder if we should let users specify the handler_class they are interested in using. That would be more flexible than having flags for everything.